projects/gallery/package.json

```diff
@@ -3,20 +3,22 @@
   "version": "17.0.0",
   "description": "A simple responsive native gallery component for Angular 17.",
   "author": "Andrey Kolkov <a.kolkov@gmail.com>",
-  "contributors": [{
-    "name": "Vincenzo Losito",
-    "url": "https://github.com/vinlos"
-  }],
+  "contributors": [
+    {
+      "name": "Vincenzo Losito",
+      "url": "https://github.com/vinlos"
+    }
+  ],
   "repository": "https://github.com/vinlos/ngx-gallery",
   "license": "MIT",
   "private": false,
   "bugs": {
     "url": "https://github.com/vinlos/ngx-gallery/issues"
   },
   "peerDependencies": {
-    "@angular/common": ">=17.3.2",
-    "@angular/core": ">=17.3.2",
-    "@angular/animations": ">=17.3.2"
+    "@angular/common": "^18.2.9",
+    "@angular/core": "^18.2.9",
+    "@angular/animations": "^18.2.9"
```

Comment:
for peer dependencies, we don't want to be so restrictive. We can then just use `>=`

Reply:
updated